src/Uno.UI/UI/Xaml/UIElement.cs

```diff
@@ -158,6 +159,12 @@ private static void OnRenderTransformOriginChanged(object dependencyObject, Depe
 		}
 		#endregion
 
+		public XamlRoot XamlRoot
+		{
+			get => _xamlRoot ?? XamlRoot.Current;
+			set => _xamlRoot = value;
```

Comment:
It's strange for me (but correct) that it's publicly settable... do you know what effect it would have to set it on UWP?

Reply:
I think I read somewhere that it is useful when there are multiple windows and you want to display for example a content dialog on a different window. If you set its `XamlRoot` to the other window's XAML root, it will be displayed there.